mne/viz/tests/test_topo.py

```diff
@@ -181,6 +181,11 @@ def test_plot_topo():
     evoked.pick_types(meg=True).plot_topo(noise_cov=cov)
     plt.close('all')
 
+    # Test exclude parameter
+    fig = picked_evoked.plot_topo(exclude=['MEG 0112'])
+    ch_count = len(picked_evoked.info['ch_names'])
+    assert len(fig.axes[0].collections) - 1 == (ch_count - 1) * 2
```

Comment:
OK, it turns out that the `LineCollection`s are the horiz/vert axes lines and the background patches; the actual data are in `fig.axes[0].lines`, so we should count that instead. Just to make this super clear for the next person who looks at it, let's assign the two things being compared here to variables with descriptive names:

```suggestion
    exclude = ['MEG 0112']
    fig = picked_evoked.plot_topo(exclude=exclude)
    n_axes_expected = len(picked_evoked.info['ch_names']) - len(exclude)
    n_axes_found = len(fig.axes[0].lines)
    assert n_axes_found == n_axes_expected
```